credoai/artifacts/model/regression_model.py

```diff
@@ -1,6 +1,15 @@
 """Model artifact wrapping any regression model"""
 from .base_model import Model
 
+from utils.model_utils import reg_handle_torch
+
+try:
```

Comment:
As it is now this will be triggered anytime a user is using the artifact and has no `torch` installed. Can it be made specific to only attempts to handle torch models without the `torch` library?